lib/system/sysio.nim

```diff
@@ -15,439 +15,5 @@
 {.push debugger:off .} # the user does not want to trace a part
```

Comment:
why not remove all code from here and put: `{.deprecated: "use `import io`".}`

Reply:
Agreed.